### PR TITLE
Added one extension to the list

### PR DESCRIPTION
**Pull request by andreseduardop**
Added Hugo Snippets to the list of Sublime Text plug-ins.

### CODE REVIEW

File: content/en/tools/editors.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ The Hugo community uses a wide range of preferred tools and has developed plug-i
 ## Sublime Text
 
 * [Hugofy](https://github.com/akmittal/Hugofy). Hugofy is a plugin for Sublime Text 3 to make life easier to use Hugo static site generator.
+* [Hugo Snippets](https://packagecontrol.io/packages/Hugo%20Snippets). Hugo Snippets is a useful plugin for add automatic snippets to Sublime Text 3. The source code can be found [here](https://github.com/regisphilibert/Sublime-Hugo-Snippets).
```
Comment (coliff) [inline content/en/tools/editors.md:26]: > plugin for add automatic snippets

Should this be 'plugin to add automatic snippets' ?

Comment (coliff) [inline content/en/tools/editors.md:26]: Would suggest removing the GitHub link. There is a link to it from https://packagecontrol.io/packages/Hugo%20Snippets anyway. Will keep things tidier.

Comment (davidsneighbour) [reply content/en/tools/editors.md:26]: "to add" or "for adding".

Comment (andreseduardop) [reply content/en/tools/editors.md:26]: "for adding", maybe?

Can you make the correction or should I create a new PR?

Comment (coliff) [reply content/en/tools/editors.md]: It's ok, I'll make the edit and merge. 👍

**Review by coliff — APPROVED** [review]

File: content/en/tools/editors.md (hunk 1 above)
Comment (andreseduardop) [reply content/en/tools/editors.md]: Thanks, @coliff

**Pull request merged**